src/api/client/resolvers.ts

```diff
@@ -29,6 +53,8 @@ export const queryNodeStitchingResolvers = (
     featuredVideos: createResolverWithoutVideoViewsField(queryNodeSchema, 'featuredVideos'),
     search: createResolverWithoutVideoViewsField(queryNodeSchema, 'search'),
```

Comment:
`search` is tricky because it deals with both videos and channels so will require both transforms to work properly. My suggestion is that we can create one function called `createResolverWithTransforms` that takes 3 params (`schema`, `fieldName` and `transforms` array), basically almost the same as what we have in `createResolverWithoutVideoViewsField`. Then here we can just pass an array of transforms for each query so it's easy to do both transform for the `search` query. Makes sense?